embedder/src/input_handling.rs

```diff
@@ -119,11 +119,40 @@ pub fn handle_input<BackendData>(event: &InputEvent<impl InputBackend>, data: &m
             }).unwrap();
         }
         InputEvent::Keyboard { event } => {
-            if event.state() != KeyState::Pressed {
-                return;
-            }
+            let glfw_keycode = keyboard::get_glfw_keycode(event.key_code());
```

Comment:
Could we get some comments to explain the process?

Reply:
I put some comments, I hope this is satisfactory. Maybe I put too many comments but I'll let you be the judge.